sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -53,361 +54,386 @@
 from apache_beam.transforms import userstate
 from apache_beam.transforms.sql import SqlTransform
 
+# Run as
+#
+# pytest flink_runner_test.py \
+#     [--test_pipeline_options "--flink_job_server_jar=/path/to/job_server.jar \
+#                               --environment_type=DOCKER"] \
+#     [FlinkRunnerTest.test_method, ...]
```

Comment:
And the test filter here doesn't work for me properly. The working version for me is 
"""
pytest flink_runner_test.py::TestClass:test_case --test-pipeline-options "--flink_job_server_jar=XXX --environment_type=XXX "
"""